journals/apps/journals/handlers.py

```diff
@@ -1,15 +1,26 @@
 from wagtail.wagtailcore.signals import page_published, page_unpublished
-from .models import JournalPage
+from .models import JournalAboutPage, JournalPage
 
 
-def pub_receiver(sender, **kwargs):
+def page_pub_receiver(sender, **kwargs):
     journal_page = kwargs['instance']
     journal_page.update_related_objects()
 
 
-def unpub_receiver(sender, **kwargs):
+def page_unpub_receiver(sender, **kwargs):
     journal_page = kwargs['instance']
     journal_page.update_related_objects(clear=True)
 
-page_published.connect(pub_receiver, sender=JournalPage)
-page_unpublished.connect(unpub_receiver, sender=JournalPage)
+def about_page_pub_receiver(sender, **kwargs):
+    journal_about_age = kwargs['instance']
+    journal_about_age.update_related_objects()
```

Comment:
shouldn't these be "journal_about_page" not "journal_about_age"?
assuming typo

Reply:
Typo, will fix